src/plugins/initiatives/initiativesMap.js

```diff
@@ -0,0 +1,32 @@
+// @flow
+
+import {type Compatible, fromCompat, toCompat} from "../../util/compat";
+import {type URL} from "./initiative";
+
+const COMPAT_INFO = {type: "sourcecred/initiatives", version: "0.1.0"};
+
+/**
+ * JSON (compat) representation of an Initiative.
+ *
+ * Major difference is that instead of using tracker addresses, we use object
+ * keys as an ID, and derive the tracker address from that.
+ */
+export type InitiativesMap = {[entryKey: string]: InitiativeEntry};
+
+type InitiativeEntry = {|
```

Comment:
propose adding `initiativeWeight: number`

then if completed === true, then we set a custom weight of `initiativeWeight`, otherwise `0`

i'm also fine with this being another PR.

Reply:
Yeah this should also be updated in the base type and createGraph, so suggest a different PR for that.